snuba/datasets/schemas/join.py

```diff
@@ -115,6 +122,11 @@ def get_columns(self) -> QualifiedColumnSet:
         column_sets = {alias: table.get_columns() for alias, table in tables.items()}
         return QualifiedColumnSet(column_sets)
 
+    def get_mandatory_conditions(self) -> Sequence[Condition]:
+        tables = self.get_tables()
+        conditions = [table.get_mandatory_conditions() for alias, table in tables.items()]
+        return chain(*conditions)
```

Comment:
I also meant to write this but forgot — `chain` returns an `Iterator`, not a `Sequence`.